qiskit/circuit/library/__init__.py

```diff
@@ -19,6 +19,54 @@
 
 .. currentmodule:: qiskit.circuit.library
```

Comment:
This file is being removed now?

Reply:
The init file? Or which one do you mean?